tests_pytest/common_tests/unit_tests/test_base_graph.py

```diff
@@ -0,0 +1,215 @@
+                                     'actq_params_fn': symmetric_selection_histogram,
+                                     'actq_method': QuantizationMethod.SYMMETRIC}
+
+        exp_actq_cfg_params_dict3 = {'quant_mode':ActivationQuantizationMode.QUANT,
+                                     'n_bits': 8,
+                                     'actq_params_fn': symmetric_selection_histogram,
+                                     'actq_method': QuantizationMethod.SYMMETRIC}
+
+        ### Expected candidates when transformed by the qcfg of FusingInfo
+        conv_1_qc_cfg = [create_exp_candidate_cfg(exp_candidate_base1, **exp_actq_cfg_params_dict1)]
+        ### Expected values when unchanged
+        relu_1_qc_cfg = [exp_candidate_base1]
+        ### Expected candidates when transformed by FusingInfo where qcfg is None
+        conv_2_qc_cfg = [create_exp_candidate_cfg(exp_candidate_base1, **exp_actq_cfg_params_dict2)]
+        ### Expected values when unchanged
+        tanh_qc_cfg   = [exp_candidate_base1]
+        ### Expected candidates with multiple configurations when transformed by the qcfg of FusingInfo
+        conv_3_qc_cfg = [create_exp_candidate_cfg(exp_candidate_base1, **exp_actq_cfg_params_dict1),
+                         create_exp_candidate_cfg(exp_candidate_base2, **exp_actq_cfg_params_dict1)]
+        ### Expected values when unchanged
+        relu_2_qc_cfg = [exp_candidate_base1, exp_candidate_base2]
+        ### Expected candidates when transformed with preserving set to True
+        flatten_qc_cfg = [create_exp_candidate_cfg(exp_candidate_base1, **exp_actq_cfg_params_dict3)]
+        ### Expected values when unchanged
+        linear_qc_cfg = [exp_candidate_base1, exp_candidate_base2]
+
+        self.exp_node_config_list = [conv_1_qc_cfg, relu_1_qc_cfg,
+                                     conv_2_qc_cfg, tanh_qc_cfg,
+                                     conv_3_qc_cfg, relu_2_qc_cfg,
+                                     flatten_qc_cfg, linear_qc_cfg]
+    
+    def check_candidates_activation_qcfg(self, candidates, exp_candidates):
+        """
+        Check if the ActivationQuantization settings set on the graph nodes match the expected values
+        """
+        for test_c, exp_c in zip(candidates, exp_candidates):
+            test_actq_cfg = test_c.activation_quantization_cfg
+            exp_actq_cfg = exp_c.activation_quantization_cfg
+            assert test_actq_cfg.quant_mode == exp_actq_cfg.quant_mode
+            assert test_actq_cfg.activation_n_bits == exp_actq_cfg.activation_n_bits
+            assert test_actq_cfg.activation_quantization_method == exp_actq_cfg.activation_quantization_method
+            assert test_actq_cfg.activation_quantization_params_fn == exp_actq_cfg.activation_quantization_params_fn
+            assert test_actq_cfg.signedness == exp_actq_cfg.signedness
+
+    def test_disable_fused_nodes_activation_quantization(self, fusing_info_generator_with_qconfig):
```

Comment:
I'm having a hard time understanding all the setup for this test.
It is a unit test, it is supposed to be small and check dedicated behaviour.
try to simplify it by using mocks for the specific functionality that is used when calling the tested function - you don't need a real graph, a real fusing info or a real tpc.